calculator/Source.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+
+#include "Calculator.h"
+#include "DivisionByZeroException.h"
+#include "UnknownOperatorException.h"
+
+
+int main() {
+
+	std::cout << "2 + 3 is " << Calculator::add(2, 3) << std::endl;
+	std::cout << "2 - 3 is " << Calculator::subtract(2, 3) << std::endl;
+	std::cout << "2 * 3 is " << Calculator::multiply(2, 3) << std::endl;
+	std::cout << "2 / 3 is " << Calculator::divide(2, 3) << std::endl;
+
+	std::cout << "2 + 3 is " << Calculator::calculate(2, '+', 3) << std::endl;
+	std::cout << "2 - 3 is " << Calculator::calculate(2, '-', 3) << std::endl;
+	std::cout << "2 * 3 is " << Calculator::calculate(2, '*', 3) << std::endl;
+	std::cout << "2 / 3 is " << Calculator::calculate(2, '/', 3) << std::endl;
+
+	try {
+		std::cout << "2 / 0 is " << Calculator::divide(2, 0) << std::endl;
+	}
+	catch (DivisionByZeroException& exception) {
```

Comment:
It's even better to catch the exception by const ref (although I like the fact you've caught it by ref and avoided an unnecessary copy).